packages/material-ui/src/styles/colorManipulator.js

```diff
@@ -126,6 +78,27 @@ export function decomposeColor(color) {
   return { type, values };
 }
 
+function intToHex(int) {
```

Comment:
Why do we need to re-arrange this? It has a very high impact on the blame of this file. This only serves people not using any form of IDE and I don't think anybody from the team has not access to code outline, no?

Reply:
It solves the `@typescript-eslint/no-use-before-define` rule

Reply:
That's weird. We had this rule before and it didn't complain. Why now?

Reply:
I assume the rule wasn't working correctly before?

Reply:
The amount of diff is off. Sending a fix.

Reply:
We had it disabled before: https://github.com/oliviertassinari/material-ui/pull/10